tests/test_BigBed_remote.py

```diff
@@ -31,8 +31,8 @@ def test_range():
     end = 10020000
     res, err = bb.getRange(chr="chr1", start=start, end=end)
     assert(err == None)
-    assert(len(res) == 437)
-    for i in range(0,437):
+    assert(len(res) == 102)
+    for i in range(0,102):
```

Comment:
pandas provides a method `iterrows` to iterate through the dataframe. That might be more appropriate here. 

I would also avoid checking explicit lengths

Reply:
the `test_range` for BigWigs also need to check if the result is within the range we query

Reply:
done